sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/README.md

```diff
@@ -26,25 +26,29 @@ Install the Azure Event Hubs client library for .NET with [NuGet](https://www.nu
 Install-Package Azure.Messaging.EventHubs.CheckpointStore.Blob -Version 1.0.0-preview.1
 ```
 
-### Obtain a connection string
+### Obtain an Event Hubs connection string
 
-For the Event Hubs client library to interact with an Event Hub, it will need to understand how to connect and authorize with it.  The easiest means for doing so is to use a connection string, which is created automatically when creating an Event Hubs namespace.  If you aren't familiar with shared access policies in Azure, you may wish to follow the step-by-step guide to [get an Event Hubs connection string](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string).
+For the event processor to interact with an Event Hub, it will need to understand how to connect and authorize with it.  The easiest means for doing so is to use a connection string, which is created automatically when creating an Event Hubs namespace.  If you aren't familiar with shared access policies in Azure, you may wish to follow the step-by-step guide to [get an Event Hubs connection string](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string).
 
-### Create an Event Processor
+### Obtain an Azure Storage connection string
 
-Once the Event Hub and connection string are available, they can be used to create an event processor for interacting with Azure Event Hubs.  The simplest way to create an `EventProcessor` is:
+For checkpoint storage to make use of Azure storage blobs, it will need to understand how to connect to a storage account and authorize with it.  The most straightforward method of doing so is to use a connection string, which is generated at the time that the storage account is created.  If you aren't familiar with storage accounts in Azure, you may wish to follow the step-by-step guide to [configure Azure Storage connection strings](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string).
+
+### Create an Event Processor that persists checkpoints in Blob Storage
+
+Once the Azure resources and connection strings are available, they can be used to create an event processor for interacting with Azure Event Hubs which persists its state via checkpoints in Azure Storage blobs.  The simplest way to create an `EventProcessor` that uses the `**TODO: CHECKPOINT BLOB NAME THING HERE**` is:
 
 ```csharp
 // CODE GOES HERE
 ```
 
 ## Key concepts
```

Comment:
This will be a follow-up set of changes, once there is actually code to inspect and extract from.  The intent of this PR is to fill in content where possible in the absence of a completed library to draw from.